pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -53,6 +54,15 @@ const (
 	// workload-bound annotations for controlling mixerfilter
 	policyCheckAnnotation = "policy.istio.io/check"
 
+	// workload-bound annotation for max retries on transport error
```

Comment:
pilot/pkg/model/context.go - in general node metadata ( like annotations ) should go there. Can move it in separate PR, I see others got in the wrong place.